Duplicati.Library.Backend.Tardigrade/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="uplink.NET" version="2.1.3" targetFramework="net462" />
```

Comment:
We should add the license information, etc. to the `Duplicati.License` project.

Reply:
I've Added the License-info